### PR TITLE
Psychologist & Lawyer stamp

### PR DESCRIPTION
**Pull request by Guess-My-Name**
## About the PR

two new stamps for better RP
added those to starting inventories as well as cmo locker (psychologist) and lawdrobe (lawyer), in case of in round promotions.
## Why / Balance

currently there's no way to make a paper impossible to change without a stamp, lawyer has to ask hop for one, psychologist has to ask hop or cmo

## Media

stampShowcase3
stampShowcase2
stampShowcase1

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

- add: Added psychologist and lawyer stamps!

### CODE REVIEW

File: Resources/Locale/en-US/paper/stamp-component.ftl [hunk 1]
```diff
@@ -15,3 +15,5 @@ stamp-component-stamped-name-warden = Warden
 stamp-component-stamped-name-trader = Trader
 stamp-component-stamped-name-syndicate = Syndicate
 stamp-component-stamped-name-ce = Chief Engineer
+stamp-component-stamped-name-lawyer = Lawyer
+stamp-component-stamped-name-psychologist = Psychologist
```
Comment (VMSolidus) [inline Resources/Locale/en-US/paper/stamp-component.ftl]: We don't need to modify upstream localization files. 

These two lines need to be moved to a Resources/Locale/en-US/DeltaV/paper/ folder(You'll have to make a folder for this). You can basically copy the stamp-component.ftl file there, and delete everything but the lawyer and psychologist localizations.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/paper/stamp-component.ftl (hunk 1 above)
Comment (Guess-My-Name) [reply Resources/Locale/en-US/paper/stamp-component.ftl]: forgot we had our own locale folder as well

**Comment by dvir001** [comment]
You can use the frontier sprites if you want
https://github.com/new-frontiers-14/frontier-station-14/pull/655
As we had the same PR not too long ago.

**Review by VMSolidus — APPROVED** [review]
This looks good now. If only stamps allowed setting a different RSI, so that we could avoid having to modify the upstream stamps. Eh that'll be something to PR to upstream anyway.

**Review by DebugOk — APPROVED** [review]
Its a shame this requires altering the upstream RSI, but it is what it is. It could be fixed at some point but eh

**Pull request merged**